components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.edit/src-gen/org/wso2/integrationstudio/gmf/esb/presentation/EEFPropertyViewUtil.java

```diff
@@ -358,6 +360,41 @@ public static ArrayList<String> getAvailableTemplateList() {
         }
         return definedTemplates;
     }
+    
```

Comment:
when writing a complete method like this we can use the correct formatting.
4 spaces instead of tas

Reply:
Fixed in : https://github.com/wso2/integration-studio/pull/1138